frontend/src/modules/entitydetails/components/GenericOriginalString.js

```diff
@@ -56,7 +56,7 @@ function getOriginalContent(props: Props) {
  * Based on the plural form, show either the singular or plural version of the
  * string, and also display which form is being rendered.
  */
-export default function GenericOriginalString(props: Props) {
+export default function GenericOriginalString(props: Props): React.Node {
```

Comment:
Exception, this might be an actual `Node` as this is more than one element. AFAICT.
Though, it seems it's an element, but I think in this case Node is more like it.